tasks.py

```diff
@@ -180,7 +181,7 @@ def validate_kind_version():
     "architecture": "CPU architecture of the local machine. Default 'amd64'.",
     "name": "name of the kind cluster to use.",
     "protocol": "Pre-configure MetalLB with the specified protocol. "
-                "Unconfigured by default. Supported: 'bgp'",
+                "Unconfigured by default. Supported: 'bgp','layer2'",
```

Comment:
I see here, it specifies "unconfigured". Maybe update the doc to reflect that.

Reply:
Update doc.